lib/tests/fairing_before_head_strip-issue-546.rs

```diff
@@ -0,0 +1,69 @@
+#![feature(plugin, decl_macro)]
+#![plugin(rocket_codegen)]
+
+extern crate rocket;
+
+use rocket::response::content;
+
+#[head("/")]
+fn index() -> content::Json<&'static str> {
+    content::Json("{ 'test': 'dont strip before fairing' }")
```

Comment:
Can we move this string into a `const` so that we don't keep repeating it?

Reply:
Also, is there a reason you're returning `Json` instead of just a `'static str`? It's better to keep it simple and focused on what we're trying to test.